proto/ibc-go-vendor/cosmos/base/tendermint/v1beta1/query.proto

```diff
@@ -0,0 +1,208 @@
+syntax = "proto3";
+package cosmos.base.tendermint.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/any.proto";
+import "google/api/annotations.proto";
+import "tendermint/p2p/types.proto";
+import "tendermint/types/types.proto";
+import "cosmos/base/query/v1beta1/pagination.proto";
+import "cosmos/base/tendermint/v1beta1/types.proto";
+import "cosmos_proto/cosmos.proto";
+import "tendermint/types/block.proto";
+
+option go_package = "github.com/cosmos/cosmos-sdk/client/grpc/tmservice";
+
+// Service defines the gRPC querier service for tendermint queries.
+// TODO(erwan): rename this back to Service, investigate the ramifications of not doing so
```

Comment:
What was the reason to rename it `TendermintQuery`? name conflicts with `tower::Service` or something?

Reply:
Yes, this is just to reduce overhead during development